web3_proxy/tests/common/create_admin.rs

```diff
@@ -38,15 +38,20 @@ pub async fn create_user_as_admin(
     };
     info!(?admin_post_login_data);
 
-    let admin_login_response = r
+    let admin_post_login_data = r
         .post(&login_post_url)
         .json(&admin_post_login_data)
         .send()
         .await
         .unwrap()
-        .json::<LoginPostResponse>()
```

Comment:
could directly read to serde_value prob (but no need to overcomplicate things)

Reply:
reading to serde_value means a 10 line long thing thats half `unwrap` just to extract the actual types. using this was way simpler.